hsc/crawler.py

```diff
@@ -10,23 +10,62 @@ class Crawler():
 	domain_url = base_url + 'domains/{}/{}'
 	problem_url = base_url + 'challenges/{}/problem'
 
-	new_readme_text = '## [{}]({})\n\nProblem Name|Problem Link|Solution Link\n---|---|---'
-	problem_readme_text = '{}|[Problem]({})|[Solution](./{}{})'
+	new_readme_text = '## [{}]({})\n\n|Problem Name|Problem Link|Solution Link|\n|---|---|---|\n'
```

Comment:
Both should work fine.
| Title1 | Title2 | Title3 |
| ------- | ------- | ------- |
OR
Title1 | Title2 | Title3
------- | -------- | -------

So, no need for this change. But, this is also okay.

Reply:
okay